dirty_cat/tests/test_docstrings.py

```diff
@@ -29,7 +29,7 @@
     "dirty_cat._minhash_encoder.MinHashEncoder",
     "dirty_cat._minhash_encoder.MinHashEncoder.fit",
     "dirty_cat._minhash_encoder.MinHashEncoder.get_fast_hash",
-    "dirty_cat._minhash_encoder.MinHashEncoder.minhash",
+    "dirty_cat._minhash_encoder.MinHashEncoder.get_murmur_hash",
```

Comment:
I think both of these (`get_fast_hash` and `get_murmur_hash`) should be private. First, they are specific to this class (no other tool in dirty_cat uses those), and second, they are parametrized by instance attributes.
I don't think we are willing to leave them in the public API for users to call.
They should however have a valid docstring (so it's necessary to leave them in this file - `test_docstrings.py`).

Reply:
Ok, you have a point. Making them both private makes sense if they are only used inside the MinHash.